content/en/community/hackathons/usoc23/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Unikraft Summer of Code 2023"
+date: 2023-04-12T01:10:00+02:00
+draft: false
+collapsible: true
+description: Unikraft Summer of Code 2023, July 3 - 15, 2023
+image: /assets/imgs/usoc23.png
+---
+
+## Unikraft Summer of Code 2023
+
+Unikraft Summer of Code 2023 (*USoC'23*) is a **free and virtual** unikernel and library operating systems workshop held by members of the Unikraft community.
+
+From zero-to-hero the two week event holds a number of starter tutorials, workshops and lessons on how to get build Unikraft unikernels for production systems.
+
+The workshop will be hands-on and will take place for 10 days, between August 29 and September 9, 2022, 4pm-7pm CEST, in English, virtually via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
```

Comment:
```suggestion
The workshop will be hands-on and will take place for 10 days, between August 29 and September 9, 2022, 4pm-7pm CEST, in English, virtually via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
```

Maybe put TODO on the dates for now?

Reply:
We know the dates, I just missed them here.